Dockerfile.tests

```diff
@@ -4,6 +4,12 @@ LABEL description="This is the build stage to create the binary."
 WORKDIR /cerenetwork
 COPY . /cerenetwork
 
+# Copy smart-contract artifacts (cere01.wasm and metadata.json)
+ARG SMART_CONTRACT_VERSION=1.1.0
+ARG ECR_REGISTRY
+COPY --from=$ECR_REGISTRY/crb-smart-contracts:$SMART_CONTRACT_VERSION /smart-contracts/cere01/target/ink/cere01.wasm /cerenetwork/frame/ddc-metrics-offchain-worker/src/tests/test_data/
```

Comment:
It is `cere02` that is needed here.

Reply:
But in README for SC : take cere01 or cere02, doesn't matter 
IS it correct ?

Because i took  cere01. Do we need cere02 ?

Reply:
Yes use cere02 everywhere.

Reply:
I know, it’s a terrible name.